docs/how-to-use-runhouse.rst

```diff
@@ -0,0 +1,285 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+When a remote object is no longer needed, it can be deallocated from
+the remote compute by calling ``cluster.delete(obj_name)``. This will remove the object from the key-value store and
+free up the memory on the worker. A worker process can similarly be terminated with ``cluster.delete(worker_name)``,
+terminating its activities and freeing its memory.
+
+To down a cluster when the task is complete and the resource is no longer needed, you can simply call ``cluster.teardown()``
+or let the autostop handle the down.
+.. code-block:: python
+
+    cluster.teardown()
+
+Moving to Production
+----------------
+A key advantage of using Runhouse is that the code developed locally has already been executing production-like on remote compute the entire time. This means
+research-to-production is a abstract checkpoint in development rather than an actual task to rewrite pipelines for production over different hardware/data.
+
+If your code is for a non-recurring task, then great, check your code into version control and you are already done. If you are deploying a recurring
+job like recurring training, then simply move the Runhouse launching code into the orchestrator or scheduler of your choice. You should not
+repackage ML code into orchestrator nodes and make orchestrators your runtime. Instead, you should use orchestrators as minimal systems to schedule and observe your jobs,
+but the jobs themselves will continue to be executed serverlessly with Runhouse from each node. This saves considerable time upfront as setting up
+the first orchestrator run less than an hour (compared to multiple weeks in traditional ML research-to-production).
+
+As an example, you might want to make the first task of your orchestrator pipeline simply bringing up the cluster and
+dispatching code to the new cluster. You can see that we are using the same underlying code (directly importing it from a source file), and then
+reusing the object and cluster by name across steps.
+
+.. code:: python
+
+      @task()
+      def up_and_dispatch():
+            cluster = rh.ondemand_cluster(
+                  name="rh-cluster",
+                  instance_type="A10G:1",
+                  provider="aws",
+                  image_id="docker:nvcr.io/nvidia/pytorch:23.10-py3",
+            ).up_if_not()
+
+            from my_code import TorchTrainer
+            my_env = rh.env(reqs=["torch"], name="my-env")
+            RemoteTrainer = rh.module(TorchTrainer).to(cluster, env=my_env)
+            trainer = RemoteTrainer(name='remote-instance-of-trainer')
+
+      @task()
+      def embed():
+            cluster = rh.cluster(name="rh-cluster")
+            trainer = cluster.get(name='remote-instance-of-trainer')
+            X, y = ...  # Load data
+            trainer.train(X,y)
+
+For production, Runhouse does recommend creating a Docker container which fixes the environment, dependencies, and program code. While
```

Comment:
dumb nit - understand the high level point but with these commas had to re-read a few times